### PR TITLE
Add titles to SVGs on build, refactor framework build scripts

### PR DESCRIPTION
**Pull request by pearofducks**

### CODE REVIEW

**Review by martin-storsletten — APPROVED** [review]

**Review by BalbinaK — APPROVED** [review]
LGTM! 
I was wondering if it makes sense to add descriptions to the alert icons that I see are in [Figma](https://www.figma.com/file/1qkEGDQWaftkOL7C360qVD/Fabric---Icon-library?node-id=3482%3A2577) too (descriptions consulted with Martin):
- AlertError16 - "Rødt utropstegn"
- AlertWarning16 - "Gult utropstegn"
- AlertInfo16 - "Info"
- AlertSuccess16 - "Grønt hake"
Screenshot 2022-08-02 at 12 09 56

File: icon-descriptions.yml [hunk 1]
```diff
@@ -4,6 +4,10 @@ AirCon: Air condition
 Airplane: Reise
 AirplaneHotel: Fly og hotell
 Alert: Utropstegn
+AlertError16: Rødt utropstegn
```
Comment (BalbinaK) [inline icon-descriptions.yml:7]: I'm not sure we should add 16 here as I think we append the size number in the import script somewhere, no?

Comment (pearofducks) [reply icon-descriptions.yml:7]: Yep you're right! My bad! 🙈

Comment (pearofducks) [reply icon-descriptions.yml:7]: Fixed on main

**Pull request merged**